ubuntuStation/views.py

```diff
@@ -289,3 +324,28 @@ def show_questions():
     #entries=[dict(questiontext=q.questiontext, questionurl=q.questionurl) for q in question]
     entries=question
     return render_template('show_questions.html', questions=entries)
+
+@app.route('/createuser', methods=['GET', 'POST'])
+@login_required
+def create_user():
```

Comment:
I am a bit confused on what is this createuser form and how it is different from the current UserForm class / page.